FhirToDataLake/deploy/templates/DeployAzureContainerApp.json

```diff
@@ -39,28 +39,46 @@
             "description": "The name of the container instance that you wish to create."
         }
     },
-    "fhirServerUrl": {
+    "dataSourceType": {
         "type": "string",
+        "defaultValue": "FHIR",
+        "allowedValues": ["FHIR", "DICOM"],
         "metadata": {
-            "description": "The fhir server endpoint you wish to get data."
+            "description": "The source API server type you wish to get data."
         }
     },
-    "authentication": {
-        "type": "bool",
-        "defaultValue": false,
-        "allowedValues": [
-        true,
-        false
-        ],
+    "fhirServerUrl": {
+        "type": "string",
+        "defaultValue": "",
         "metadata": {
-            "description": "Whether to access Azure API for FHIR with managed identity authentication. Set false if you are using an OSS public FHIR server instance."
+            "description": "The fhir server endpoint you wish to get data."
         }
     },
     "fhirVersion": {
         "type": "string",
         "defaultValue": "R4",
```

Comment:
do we need to allow an empty value for Dicom?

Reply:
I feel empty value may also be set up to fhirToDatalake pipeline, which bings a possible error operation. 

In fact we will set up the environment variable of Container App to empty if **dataSourceType** is DICOM